app/jobs/assignment_repo/create_github_repository_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class AssignmentRepo
+  class CreateGitHubRepositoryJob < ApplicationJob
+    queue_as :create_repository
+    retry_on Creator::Result::Error, queue: :create_repository
```

Comment:
`retry_on` defaults to 3s wait and 5 attempts. 
This should be enough for what we are trying to do, right?

Also if we wanted we could pass a `wait: :exponentially_longer` which would perform an exponential backoff algorithm.

Reply:
I'd vote for the exponential backoff.

Reply:
Yeah, especially because the main reason we'd hit a problem like this is GitHub API downtime / issues.

Reply:
I'll add 🚩